docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
     command: mkcert -cert-file "${DEV_URL:-local.devgo.vip}.crt" -key-file "${DEV_URL:-local.devgo.vip}.key" "${DEV_URL:-local.devgo.vip}" "*.${DEV_URL:-local.devgo.vip}"
 
   db:
-    image: mysql:5.7
+    image: mariadb:10.5.8
```

Comment:
The VIP dev environment appears to be using version 10.3 per https://github.com/Automattic/vip/blob/c011c453054d197a69a3502ce7f58c043e728a8d/src/lib/constants/dev-environment.js#L8

I wonder if we should stick to that for now, too?

Reply:
Sure, that makes sense!